### PR TITLE
Fix issue 37154: samesite description

### LINKED ISSUE #37154: SameSite behaviour incorrectly described in the Security/Practical Implementation Guide for cookies
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Cookies

### What specific section or headline is this issue about?

SameSite description does not match with "Using HTTP Cookies"

### What information was incorrect, unhelpful, or incomplete?

The Same-Site description lists that Cookies are omitted in same-origin contexts (e.g. navigating a.example.com to b.example.com). This conflicts with the information in the [Cookies MDN page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#controlling_third-party_cookies_with_samesite) and in [PortSwigger's SameSite blog post](https://portswigger.net/web-security/csrf/bypassing-samesite-restrictions). 

### What did you expect to see?

I would expect for this behaviour to not be listed as an example.

### Do you have any supporting links, references, or citations?

- https://portswigger.net/web-security/csrf/bypassing-samesite-restrictions
- https://gist.github.com/oscarychen/4a6568da34817fc2ba98a5068da5ec70#cookie-attributes
- https://security.stackexchange.com/a/223477
- https://www.thinktecture.com/en/identityserver/samesite-cookies-in-a-nutshell/

### Do you have anything more you want to share?

N/A

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/security/practical_implementation_guides/cookies`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Cookies
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/security/practical_implementation_guides/cookies/index.md
* Last commit: https://github.com/mdn/content/commit/5c000c8621145c6915f3d545b505c216317bc64a
* Document last modified: 2024-07-26T02:23:10.000Z

</details>

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/37154.

### CODE REVIEW

File: files/en-us/web/security/practical_implementation_guides/cookies/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ To minimize the scope for cookie vulnerabilities on your site, limit access to c
 
   - : Forbid sending cookies via cross-origin requests (for example from {{htmlelement("img")}} elements) using `SameSite`. You should use one of the following two values:
 
-    - `SameSite=Strict`: Only send the cookie in same-site contexts (navigations and other requests). Cookies are omitted in same-origin contexts (e.g. navigating `a.example.com` to `b.example.com`), cross-site requests (e.g. hotlinking), and cross-site navigation (e.g. when following a link from a different web page). This is a very strict setting, but it does provide strong [CSRF](/en-US/docs/Web/Security/Practical_implementation_guides/CSRF_prevention) protection, so use this value if possible.
+    - `SameSite=Strict`: Only send the cookie in same-site contexts (navigations and other requests). Cookies are omitted in cross-site requests (e.g. hotlinking) and cross-site navigation (e.g. when following a link from a different web page). This is a very strict setting, but it does provide strong [CSRF](/en-US/docs/Web/Security/Practical_implementation_guides/CSRF_prevention) protection, so use this value if possible.
```
Comment (hamishwillee) [inline files/en-us/web/security/practical_implementation_guides/cookies/index.md:40]: I feel ignorant, but what is hotlinking? Do you mean

```suggestion
    - `SameSite=Strict`: Only send the cookie in same-site contexts (navigations and other requests). Cookies are omitted in cross-site requests (e.g. embedding images or other resources from other sites) and cross-site navigation (e.g. when following a link from a different web page). This is a very strict setting, but it does provide strong [CSRF](/en-US/docs/Web/Security/Practical_implementation_guides/CSRF_prevention) protection, so use this value if possible.
```

**Review by hamishwillee — APPROVED** [review]
This is clearly correct, but one query/suggestion here: https://github.com/mdn/content/pull/37189/files#r1883166075

Approving so you can merge when you have had a look at that.

File: files/en-us/web/security/practical_implementation_guides/cookies/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/security/practical_implementation_guides/cookies/index.md:40]: I didn't write this guide but I believe so.

**Comment by wbamberg** [comment]
Thanks! I think there are potentially more issues and improvements to the page but in the current PR I only wanted to fix this, which AFAICS is just an error.

**Pull request merged**